test/main.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
 
+var concatStream = require('concat-stream');
 var replacePlugin = require('../');
 var fs = require('fs');
-var es = require('event-stream');
 var should = require('should');
 var File = require('vinyl');
-require('mocha');
```

Comment:
Actually, this pull request passes the test . https://travis-ci.org/lazd/gulp-replace/builds/69782395